code/transportation-development-services/tds.js

```diff
@@ -472,14 +408,120 @@ function showFormFieldsByValue(value) {
   });
 });
+
+/**************************************/
+/*** Redirect from Blank Menu Pages ***/
+/**************************************/
+//Development Reviews Menu
+$(document).on('knack-scene-render.scene_28', function(event, scene) { 
+window.location.href = "https://atd.knack.com/development-services#development-reviews/";
+});
+
+//TIA Reviews Menu
+$(document).on('knack-scene-render.scene_108', function(event, scene) { 
+window.location.href = "https://atd.knack.com/development-services#tia-requests/";
+});
+
+//TDA Reviews Menu
+$(document).on('knack-scene-render.scene_260', function(event, scene) { 
+window.location.href = "https://atd.knack.com/development-services#tda-reviews/";
+});
+
+//Customer Portal Menu
+$(document).on('knack-scene-render.scene_137', function(event, scene) { 
+window.location.href = "https://atd.knack.com/development-services#customer-home/";
+});
+
+//Help Menu
+$(document).on('knack-scene-render.scene_240', function(event, scene) { 
+window.location.href = "https://atd.knack.com/development-services#customer-help/";
+});
+
+/************************************/
+/**** Save TIA Request Record ID ****/
+/************************************/
+// Function to Save Record ID
+function save_Record_ID(recordID) {
+  $.ajax({
+    url: "https://api.knack.com/v1/pages/scene_267/views/view_732/records/" + recordID, // Scene/View of Second Form
+    type: "PUT", 
+    headers: {
+      "X-Knack-Application-Id": Knack.application_id,
+      "X-Knack-REST-API-Key": `knack`,
+      "Authorization": Knack.getUserToken(),
+    },
+    data: {
+      field_824 : recordID, // Store Record ID in Knack Record ID field on TIA Request object
+    },
+    tryCount: 0,
+    retryLimit: 3,
+    success: function(response) {
+      console.log("Captured Record ID"); // Success Message in Console Log
+      Knack.hideSpinner();
+    },
+    error : function(XMLHttpRequest, textStatus, errorThrown) {
+      this.tryCount++;
+      let tryCount = this.tryCount, retryLimit = this.retryLimit, seconds; 
+      if (tryCount <= retryLimit) { //try again
+        switch(tryCount) {
+          case 1:
+          case 2: seconds = 5; break;
+          case 3: seconds = 10; break; }
+        let timeout = seconds * 1000;
+        console.log("Error: " + XMLHttpRequest.status + " " + XMLHttpRequest.statusText + "\nRetry Count: " + tryCount + "\nRetrying in " + seconds + " seconds")
+        let ajaxObject = this;
+        window.setTimeout(function(){
+          $.ajax(ajaxObject);
+        }, timeout);
+      } else {
+        console.log("Failed to Capture Record ID");
+      }
+    }
+  });
+}
+
+// Listen for record creation (Request Application Submit / First Form)
+// The comment line below also works
+//$(document).on('knack-form-submit.view_731' , function(event, view, record) {
+$(document).on('knack-record-create.view_731', function(event, view, record) {
+  const recordId = record.id;
+  save_Record_Id(recordId);
```

Comment:
change to `save_Record_ID(recordId)` 
the function name doesnt match, the code sample led us astray!!

Reply:
Great catch, @chiaberry! Might be worth updating both the definition and call to `saveRecordId` to stick to the JavaScript naming convention. Glad this is working!